### ISSUE TITLE
Participate now button on study detail view not in tab order or labelled

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#702]
On the Baby Tetris study page, the Participate button is not labeled and is not in the tab order so not available to keyboard only users. See https://github.mit.edu/Accessibility-FY2021/lookit/issues/16

**Comment by okaycj** [comment]
The referenced issue has been closed.
